packages/libs/coreui/src/components/Mesa/Ui/HeadingCell.jsx

```diff
@@ -186,14 +186,14 @@ class HeadingCell extends React.PureComponent {
 
     if (!column.helpText && !column.htmlHelp) return null;
     return (
-      <Tooltip
+      <MesaTooltip
```

Comment:
I swapped out this instance of `MesaTooltip` with `HelpTrigger` as shown below, and things seem to work. May need more testing, though.

```js
<HelpTrigger>
  {column.htmlHelp ?? column.helpText}
</HelpTrigger>
```